src/test/e2e/bundle_test.go

```diff
@@ -108,138 +110,143 @@ func TestBundle(t *testing.T) {
-	zarfPublish(t, pkg, "localhost:889")
-
-	bundleDir := "src/test/bundles/01-uds-bundle"
-	bundlePath := filepath.Join(bundleDir, fmt.Sprintf("uds-bundle-example-%s-0.0.1.tar.zst", e2e.Arch))
+	bundleDir := "src/test/bundles/03-local-and-remote"
+	bundlePath := filepath.Join(bundleDir, fmt.Sprintf("uds-bundle-test-local-and-remote-%s-0.0.1.tar.zst", e2e.Arch))
 
 	createLocal(t, bundleDir, e2e.Arch)
-	inspectLocal(t, bundlePath)
-	inspectLocalAndSBOMExtract(t, bundlePath)
-
-	// Test only podinfo deploy
-	deployPackagesFlag(bundlePath, "podinfo")
-	cmd := strings.Split("zarf tools kubectl get deployments -A -o=jsonpath='{.items[*].metadata.name}'", " ")
-	deployments, _, _ := e2e.UDS(cmd...)
-	require.Contains(t, deployments, "podinfo")
-	require.NotContains(t, deployments, "nginx")
-
-	remove(t, bundlePath)
-
-	// Test both podinfo and nginx deploy
-	deployPackagesFlag(bundlePath, "podinfo,nginx")
-	deployments, _, _ = e2e.UDS(cmd...)
-	require.Contains(t, deployments, "podinfo")
-	require.Contains(t, deployments, "nginx")
-
-	// Remove only podinfo
-	removePackagesFlag(bundlePath, "podinfo")
-	deployments, _, _ = e2e.UDS(cmd...)
-	require.NotContains(t, deployments, "podinfo")
-	require.Contains(t, deployments, "nginx")
-
-	// Remove nginx
-	removePackagesFlag(bundlePath, "nginx")
-	deployments, _, _ = e2e.UDS(cmd...)
-	require.NotContains(t, deployments, "podinfo")
-	require.NotContains(t, deployments, "nginx")
 
-	// Test invalid package deploy
-	_, stderr := deployPackagesFlag(bundlePath, "podinfo,nginx,peanuts")
-	require.Contains(t, stderr, "invalid zarf packages specified by --packages")
+	t.Run("Test only podinfo deploy (local pkg)", func(t *testing.T) {
+		deployPackagesFlag(bundlePath, "podinfo")
+		cmd := strings.Split("zarf tools kubectl get deployments -A -o=jsonpath='{.items[*].metadata.name}'", " ")
+		deployments, _, _ := e2e.UDS(cmd...)
+		require.Contains(t, deployments, "podinfo")
+		require.NotContains(t, deployments, "nginx")
+		remove(t, bundlePath)
+	})
 
-	// Test invalid package remove
-	_, stderr = removePackagesFlag(bundlePath, "podinfo,nginx,peanuts")
-	require.Contains(t, stderr, "invalid zarf packages specified by --packages")
+	cmd := strings.Split("zarf tools kubectl get deployments -A -o=jsonpath='{.items[*].metadata.name}'", " ")
+	t.Run("Test only nginx deploy (remote pkg)", func(t *testing.T) {
+		deployPackagesFlag(bundlePath, "nginx")
+		deployments, _, _ := e2e.UDS(cmd...)
+		require.Contains(t, deployments, "nginx")
+		require.NotContains(t, deployments, "podinfo")
+		remove(t, bundlePath)
+	})
+
+	t.Run("Test both podinfo and nginx deploy", func(t *testing.T) {
+		deployPackagesFlag(bundlePath, "podinfo,nginx")
+		deployments, _, _ := e2e.UDS(cmd...)
+		require.Contains(t, deployments, "podinfo")
+		require.Contains(t, deployments, "nginx")
+
+	})
+	t.Run("Remove only podinfo (local pkg)", func(t *testing.T) {
+		removePackagesFlag(bundlePath, "podinfo")
```

Comment:
should we deploy the packages again or make sure that they are deployed before trying to run this test. (removing dependencies from previous test that does the deployment)